### PR TITLE
add Shoebill token 🐦

### PR DESCRIPTION
**Pull request by leafwind**
- [x] I will not ping the Discord about this pull request.

Edit: the official page was updated!

## Purpose

shoebill_250

Shoebill Coin ($SHBL) is a memecoin built on Solana blockchain. The purpose of building Shoebill Coin is to acquire people's attention to the Solana blockchain, which is convenient, cheap and fast.

Also $SHBL can make more people know about this cute, critically endangered species: Shoebill is mainly found in eastern-central Africa. Shoebills population ranges from 5,000 to 8,000. Bird Life International has classified them as vulnerable species.

WARNING:
- You are encouraged to collect and share to your friend, but it does not give any right of return, management, governance, and any other right.
- You should NOT treat it as an investment target, and any trading results from price fluctuation is considered as speculation.

### Tokenomics & Plan
- Max supply: fixed 10,000,000,000 SHBL
- Airdrop: planning, will mainly on Taiwan blockchain Telegram groups
- Burn: TBD (but for an educational purpose, tend to burn 90%)
- Stickers：$SHBL is collaborating with a painter who loved birds for a long time, we are going to introduce Line sticker, and will also be available on Telegram.

### Market information
- I have add the market to Serum, so it is technically tradable. (market ID: `9G2bAA5Uv8JyPZteuP73GJLUGg5CMbhMLCRSBUBLoXyt`)
- I also deployed a [Serum-forked frontend](http://shoebill-serum.leafwind.tw/#/market/9G2bAA5Uv8JyPZteuP73GJLUGg5CMbhMLCRSBUBLoXyt) to demonstrate that, which is very primitive: no SSL, no promising change but works.

### Minimum information of SHBL token:
* Token Address: 7fCzz6ZDHm4UWC9Se1RPLmiyeuQ6kStxpcAP696EuE1E
* Token Name: Shoebill Coin
* Token Symbol: SHBL
* Logo: https://cdn.jsdelivr.net/gh/leafwind/shoebill-coin/shoebill.png
* Link to the official homepage of token: https://leafwind.tw/shoebill-coin/
* Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_): TBD

### CODE REVIEW

**Comment by leafwind** [comment]
resolved conflict again.

**Comment by leafwind** [comment]
remove unexpected diff during resolving conflicts

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 0.2.97 (2021-05-15)
```
Comment (leafwind) [inline CHANGELOG.md:5]: This was automatically bumped, please directly modify it if needed, thanks 🙇

**Comment by princeshun** [comment]
Nice

**Comment by leafwind** [comment]
@bartosz-lipinski sorry to tag you specifically, I haven't receive any suggestions so far, but if anything I missed, please remind me, I will make the change ASAP. 
Appreciate you contribution 🙇

**Pull request merged**